examples/with-sentry/components/withSentry.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Raven from 'raven-js'
+
+function withSentry (Child) {
+  return class WrappedComponent extends React.Component {
+    static getInitialProps (context) {
+      return Child.getInitialProps(context)
```

Comment:
Check if `Child.getInitialProps` exists before running it, a Page could not have a `getInitialProps` method

Reply:
Done in latest commit.